src/TODO

```diff
@@ -1,103 +0,0 @@
-TODO
```

Comment:
This File is completely outdated, so I suggest to remove it and use Githubs issuetracker and pull requests to manage open topics